DemoApp/StreamChat/Components/DemoChatChannelVC.swift

```diff
@@ -17,7 +18,40 @@ final class DemoChatChannelVC: ChatChannelVC, UIGestureRecognizerDelegate {
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-    
+
+    // MARK: - Custom Header Views
+
+    // Example of custom header views.
+    // The same code should be provided to `ChatThreadVC` if you want the same behaviour in threads.
+
+    lazy var messageListHeaderView: UIView? = {
```

Comment:
maybe the loading indicator is enough for the showcase? I think we should keep the sample app as clean as possible, and this label is not very common for messaging apps.

Reply:
We can explain in the docs the `channelController.hasLoadedAllPreviousMessages`, customers here would usually show some custom UI.

Reply:
Yeah, I was planning on adding docs. So yeah maybe the "Loaded all messages" we can do it only for the docs 👍